components/gitpod-cli/cmd/tasks-stop.go

```diff
@@ -118,12 +113,12 @@ var stopTaskCmd = &cobra.Command{
 		}
 
 		for _, terminalAlias := range terminalAliases {
-			_, err := client.Terminal.Shutdown(context.Background(), &api.ShutdownTerminalRequest{Alias: terminalAlias})
+			_, err = client.Terminal.Shutdown(ctx, &api.ShutdownTerminalRequest{Alias: terminalAlias})
```

Comment:
@iQQBot this is another command which should be reported before calling? i.e. if we are in terminal which is about to be stopped